resources/styles/book-content/target.less

```diff
@@ -0,0 +1,61 @@
+.highlight-animation() {
+  -webkit-animation: highlight 1.5s ease-out;
+  animation: highlight 1.5s ease-out;
+}
+
+.offset-header() {
+  margin-top: -@tutor-navbar-height;
+  padding-top: @tutor-navbar-height;
+}
+
+a[target=_blank] {
+  &::after {
+    .fa-icon();
+    content: @fa-var-external-link;
+    margin: 0 4px 0 6px;
+  }
+}
+
+figure{
+  &:target, .target {
+    .offset-header();
+    .highlight-animation();
+
+    > * {
+      .highlight-animation();
+    }
+
+    img {
+      mix-blend-mode: multiply;
+    }
+  }
+}
+
+section, div, span, table {
+  &:target, .target {
+    .offset-header();
+    .highlight-animation();
+
+    img {
+      mix-blend-mode: multiply;
+    }
+  }
+}
+
+@-webkit-keyframes highlight {
+  0% {
+    background-color: @tutor-highlight;
+  }
+  100% {
+    background-color: transparent;
+  }
+}
+
+@keyframes highlight {
```

Comment:
Since keyframes are global can this be something like `targetHighlight` that way it does not conflict or move it? so it is used in the iReading as well as the book reader?